<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>水印消除工具 V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/myIcons/icon/logo.png</normaloff>:/myIcons/icon/logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="LoadImageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>原始图片列表</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="LoadImagePathLayout">
          <item>
           <widget class="QPushButton" name="LoadImagePathButton">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;返回上一级目录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;返回上一级目录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="myresources.qrc">
              <normaloff>:/myIcons/icon/up.png</normaloff>:/myIcons/icon/up.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LoadImagePathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="LoadImageListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SaveImageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>保存图片列表</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="SaveImagePathLayout">
          <item>
           <widget class="QPushButton" name="SaveImagePathButton">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;返回上一级目录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="myresources.qrc">
              <normaloff>:/myIcons/icon/up.png</normaloff>:/myIcons/icon/up.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SaveImagePathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="SaveImageListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1,1,1,1,1">
          <item>
           <widget class="QLabel" name="OriImageLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>原始视图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ForeColorInfoLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>前景色：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="ForeColorLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="BackColorInfoLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>背景色：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="BackColorLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="CurrentImageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>操作视图</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MyGraphicsView" name="OriImageGraphicsView">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MyGraphicsView" name="CurrentImageGraphicsView">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ImageResultLayout" stretch="0,0,0,0">
        <item>
         <widget class="QLabel" name="MaskImageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>预提取水印样式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="MaskImageGraphicsView"/>
        </item>
        <item>
         <widget class="QLabel" name="ResultImageLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模拟去除结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="ResultImagegraphicsView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="StatusInfoLayout" stretch="1,2">
      <item>
       <layout class="QHBoxLayout" name="StatusLayout">
        <item>
         <widget class="QLabel" name="LocationLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PixelLocationLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SizeLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>图片大小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ImageSizeLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>false</underline>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="Recall"/>
    <addaction name="recovery"/>
    <addaction name="separator"/>
    <addaction name="settings"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="ToolMenu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="Pencil"/>
    <addaction name="Eraser"/>
    <addaction name="Glasses"/>
    <addaction name="Hand"/>
    <addaction name="separator"/>
    <addaction name="RectSelect"/>
    <addaction name="FreeSelect"/>
   </widget>
   <widget class="QMenu" name="AboutMenu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QMenu" name="WindowMenu">
    <property name="title">
     <string>窗口</string>
    </property>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="ToolMenu"/>
   <addaction name="WindowMenu"/>
   <addaction name="AboutMenu"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Open"/>
   <addaction name="settings"/>
   <addaction name="Save"/>
   <addaction name="Recall"/>
   <addaction name="recovery"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Pencil"/>
   <addaction name="Eraser"/>
   <addaction name="Glasses"/>
   <addaction name="Hand"/>
  </widget>
  <widget class="QToolBar" name="selectToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="RectSelect"/>
   <addaction name="FreeSelect"/>
  </widget>
  <widget class="QToolBar" name="extraToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="GetColor"/>
  </widget>
  <action name="Open">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/open.png</normaloff>:/myIcons/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开图片文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="Save">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/save.png</normaloff>:/myIcons/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Recall">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/back.png</normaloff>:/myIcons/icon/back.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="Exit">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/exit.png</normaloff>
     <normalon>icon/exit.png</normalon>:/myIcons/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="Pencil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/pencil.png</normaloff>:/myIcons/icon/pencil.png</iconset>
   </property>
   <property name="text">
    <string>铅笔</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="Eraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/eraser.png</normaloff>:/myIcons/icon/eraser.png</iconset>
   </property>
   <property name="text">
    <string>橡皮</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="Glasses">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/glasses.png</normaloff>:/myIcons/icon/glasses.png</iconset>
   </property>
   <property name="text">
    <string>放大镜</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="Hand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/hand.png</normaloff>:/myIcons/icon/hand.png</iconset>
   </property>
   <property name="text">
    <string>抓手工具</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="RectSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/rece.png</normaloff>:/myIcons/icon/rece.png</iconset>
   </property>
   <property name="text">
    <string>矩形选择工具</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="FreeSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/freeSelect.png</normaloff>:/myIcons/icon/freeSelect.png</iconset>
   </property>
   <property name="text">
    <string>自由选择工具</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="recovery">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/forward.png</normaloff>:/myIcons/icon/forward.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/config.png</normaloff>:/myIcons/icon/config.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="GetColor">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myIcons/icon/color.png</normaloff>:/myIcons/icon/color.png</iconset>
   </property>
   <property name="text">
    <string>编辑颜色</string>
   </property>
   <property name="toolTip">
    <string>编辑颜色</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
